src/components/PaymentStripe/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { loadStripe } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+import { useLocale } from '../../libs/hooks';
+import CheckoutForm from './checkoutForm';
+
+/**
+ * Payment Stripe component are used for freeform data entry.
+ * @category Form
+ */
+const PaymentStripe = props => {
+    const { apiKeyStripe, className, style, ...rest } = props;
+    const locale = useLocale();
+    const elementOptions = { locale };
```

Comment:
we can memo this value here as well, if not it will pass a different object `options` every time the function runs